dashboard/src/components/ChartList/ChartListItem.css

```diff
@@ -3,34 +3,34 @@
 }
 
 .ChartListItem__content {
-  display: flex;
-  justify-content: space-between;
 }
 
 .ChartListItem__content__title {
-  /* Avoid 3 line elements */
-  word-break: break-all;
-  flex: 1;
-  font-size: 1.1em;
+  color: black;
```

Comment:
Could you update this color to the text one? `#1C2B39` https://design.bitnami.com/category/Foundations/Variables/index.html#Colors